### PR TITLE
Changed Spacebar hotkey to be in a mvvm pattern

### PR DESCRIPTION
**Pull request by mreichardt95**
No more Code behind.

### CODE REVIEW

File: Source/TailBlazer/Views/Tail/ActionAutoTail.cs [hunk 1]
```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Input;
+
+namespace TailBlazer.Views.Tail
+{
+    class ActionAutoTail : ICommand
```
Comment (RolandPheasant) [inline Source/TailBlazer/Views/Tail/ActionAutoTail.cs]: Do not need this as Command already does it

i.e. var  command =  new Command(() =>//...Do some action);

File: Source/TailBlazer/Views/Tail/TailViewModel.cs [hunk 2]
```diff
@@ -244,6 +247,18 @@ public class TailViewModel: AbstractNotifyPropertyChanged, ILinesVisualisation,
         {
            await DialogHost.Show(SearchOptions, Id);
         }
+
```
Comment (RolandPheasant) [inline Source/TailBlazer/Views/Tail/TailViewModel.cs]: For consistency change to this 

ICommand KeyAutoTail {get;} = new Command(() => { AutoTail = true;});

**Comment by mreichardt95** [comment]
Thanks for the tips! :)

**Comment by RolandPheasant** [comment]
Done

**Pull request merged**